### PR TITLE
Blog - Deploying Docker Containers with ECS

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+Firstly, we need to create a virtual environment and install our dependencies.
+
+~~~{.bash caption=">_"}
+virtualenv env
+~~~
+
+~~~{.bash caption=">_"}
+pip install django djangorestframework
+~~~
```
Comment (jalletto) [inline blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:60]: When two separate commands are on top of each other like this, they tend to blend together. We can seperate them by adding a -
```suggestion

~~~{.bash caption=">_"}
virtualenv env
~~~

-

~~~{.bash caption=">_"}
pip install django djangorestframework
~~~
```

Comment (jalletto) [reply blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:60]: When there are two seperate code examples on back to back with nothing inbetween, they will blend together in the doc. You can put a single blank line between them by adding a `-`.

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+Next, we need to activate our virtual environment and create our Django project.
+
+~~~{.bash caption=">_"}
+source env/bin/actviate
+~~~
+
+~~~{.bash caption=">_"}
+django-admin startproject app .
+~~~
+
```
Comment (jalletto) [inline blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:71]:
```suggestion

~~~{.bash caption=">_"}
source env/bin/actviate
~~~

-

~~~{.bash caption=">_"}
django-admin startproject app .
~~~

```

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+<div class="wide">
+
+![Image]({{site.images}}{{page.slug}}/image20.png)\
+
+</div>
+
+As you can see, we got a response of {"message": "Hello World"}.
+
+With that, we have just successfully deployed our docker container.
```
Comment (jalletto) [inline blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:423]:
```suggestion
</div>

As you can see, we got a response of `{"message": "Hello World"}`.

With that, we have just successfully deployed our docker container.
```

**Review by jalletto — CHANGES_REQUESTED** [review]
Just some small changes for this one.

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md (hunk 3 above)
Comment (msali123) [reply blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:423]: Done

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md (hunk 2 above)
Comment (msali123) [reply blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:71]: Done

File: blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2022-11-03-deploy-DockerContainers-with-ECS.md:60]: Done

**Comment by jalletto** [comment]
@msali123 Never mind about the -. I put them in the other PR too, but you can ignore that and just leave it as is. I think this is something we need to change in the CSS. My mistake.

**Review by jalletto — APPROVED** [review]

**Pull request merged**